app/code/Magento/Backend/Block/Media/Uploader.php

```diff
@@ -79,6 +89,16 @@ public function getFileSizeService()
         return $this->_fileSizeService;
     }
 
+    /**
+     * Get image config
+     *
+     * @return \Magento\Framework\Image\Adapter\ConfigInterface
+     */
+    public function getImageConfigService()
+    {
+        return $this->imageConfig;
```

Comment:
Such method must be eliminated as it violates Law of Demeter, inject needed classes into blocks directly and introduce methods like `getImageMaxWidth`.